client/src/globals.js

```diff
@@ -31,3 +31,10 @@ export function getUsers() {
   return users
 }
 
+// Gets called onKeyDown from textArea of form element, if keys pressed are ctrl+enter or
+//  metaKey+enter, clicks parentelements input element which has attribute type="submit"
+export function keyPressSubmit(e) {
```

Comment:
Would it be better to give this event handler the submit button as an argument? It would be purer this way

Reply:
ok ok